apa-annotated-bibliography.csl

```diff
@@ -679,7 +684,7 @@
   </macro>
   <citation et-al-min="6" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year" givenname-disambiguation-rule="primary-name">
     <sort>
-      <key macro="author"/>
+      <key macro="author" names-min="8" names-use-first="6"/>
```

Comment:
@adam3smith, I assume you checked the rules here, but isn't it a bit strange that the sorting rules differ from the number of names shown?

Reply:
Manual says explicitly same order in text as in the bibliography (and I recall vaguely that the reason this came up is a mis-ordered 2001a/2001b